<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dev2</class>
 <widget class="QMainWindow" name="Dev2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dev2</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>&amp;Database</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionIP_Addresses"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>D&amp;ata</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionIP_Networks"/>
    <addaction name="actionDevice_Types"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout_QT"/>
   </widget>
   <addaction name="menuDatabase"/>
   <addaction name="menuData"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCreate">
   <property name="text">
    <string>&amp;Create</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>&amp;Import</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>&amp;Export</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionInsert">
   <property name="text">
    <string>&amp;Insert</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="text">
    <string>&amp;Update</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>&amp;Delete</string>
   </property>
  </action>
  <action name="actionAbout_QT">
   <property name="text">
    <string>&amp;About QT</string>
   </property>
  </action>
  <action name="actionAttributes">
   <property name="text">
    <string>&amp;Attributes</string>
   </property>
  </action>
  <action name="actionIP_Networks">
   <property name="text">
    <string>&amp;IP Networks</string>
   </property>
  </action>
  <action name="actionIP_Addresses">
   <property name="text">
    <string>&amp;IP Addresses</string>
   </property>
  </action>
  <action name="actionDevice_Types">
   <property name="text">
    <string>Device Types</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
